esp/scm/ws_dfu.ecm

```diff
@@ -707,34 +707,46 @@ ESPresponse [exceptions_inline, nil_remove, http_encode(0)] DFUGetFileMetaDataRe
     int64 TotalResultRows;
 };
 
-
 ESPrequest ListHistoryRequest
 {
-    string name;
+    string Name;
+};
+
+ESPStruct History
+{
+    string Name;
+    string Operation;
+    string Timestamp;
+    string IP;
+    string Path;
+    string Owner;
+    string Workunit;
 };
 
 ESPresponse [exceptions_inline, nil_remove] ListHistoryResponse
 {
-    binary xmlmap;
+    [depr_ver("1.36")] binary xmlmap;
+    ESParray<ESPStruct History> FileHistory;
```

Comment:
oh ok, the inconsistency has to be addressed, I thought you were pointing out the element names weren't optimal.

If you're ok with `ESParray<ESPStruct History> FileHistory;` then let's keep it, otherwise one of these two might be more appropriate:
`ESParray<ESPStruct Operation> Operations;` or
`ESParray<ESPStruct Operation> History;`